.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,51 @@
+name: Run markdownlint
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+      - '!translated/**'
+      - '!versions/**'
+  workflow_dispatch: # Manual run
+
+concurrency:
+  group: markdownlint-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  markdownlint:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - id: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - id: changes
+        uses: tj-actions/changed-files@v44
+        with:
+          files: |
+            **/*.md
+            !translated/**
+            !versions/**
+
+      - id: node
+        if: steps.changes.outputs.any_changed == 'true'
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+        continue-on-error: true  # lint even without `titlecase`
+
+      - id: titlecase
+        if: steps.node.outcome == 'success'
+        run: npm install markdownlint-rule-titlecase@v0.1.0
```

Comment:
What does this do? is this magically picked up by the next step?

Reply:
To make sure that all headings use Title Case, I used a custom `markdownlint` rule.

> is this magically picked up by the next step?

See <https://github.com/its-miroma/fabric-docs/blob/e1c496ae37812f77bfa68cba9b4419af352c3af2/.markdownlint-cli2.yaml#L50-L51>